workflows/explore_eml/navigate_through_eml.Rmd

```diff
@@ -5,12 +5,13 @@ The first task when editing an EML file is navigating the EML file. An EML file
 ```{r echo=FALSE, message = FALSE, error = FALSE}
 library(dataone)
 library(EML)
-
-cn <- CNode('PROD')
-adc <- dataone::getMNode(cn, 'urn:node:ARCTIC')
 ```
 
 ```{r}
+# Need to be in this member node to explore file
+cn_staging <- CNode('STAGING')
+adc_test <- getMNode(cn_staging,'urn:node:mnTestARCTIC')
```

Comment:
Ah, you're correct, does not work. I had assumed the test site would have everything the production site does. Concepts of prod/test are introduced, I was just thinking that if someone who is a complete stranger to this sort of thing was going through the training they could still make a mistake. Originally the code pointing to the production node was in an invisible chunk but that made it impossible to replicate on your own machine w/out looking at the file as an rmd - does prevent someone from writing in prod but they can't actually follow this part of training on own machine. So - could upload the dataset to the test node so that this code would work, use visible production node with extra warnings, or keep as is?

Reply:
Let's upload the dataset to the test site (it will need a different pid, there is a function `clone_package` that does this) and change it to the test node. Can you do this? Let me know if you have questions/issues

Reply:
Looks like the book build is also failing in this section but hopefully your fix will resolve that issue

```
label: unnamed-chunk-48
Quitting from lines 11-16 (./../workflows/explore_eml/navigate_through_eml.Rmd) 
Quitting from lines NA-32 (./../workflows/explore_eml/navigate_through_eml.Rmd) 
Error in getObject(adc, "doi:10.18739/A2FS1H") : object 'adc' not found
Calls: <Anonymous> ... eval -> eval -> read_eml -> <Anonymous> -> getObject
```